packages/babel-core/package.json

```diff
@@ -32,7 +32,7 @@
     "babel-template": "7.0.0-alpha.15",
     "babel-traverse": "7.0.0-alpha.15",
     "babel-types": "7.0.0-alpha.15",
-    "babylon": "7.0.0-beta.15",
+    "babylon": "7.0.0-beta.17",
```

Comment:
👍 , you'll need to add it to a few more places (you can just search `"babylon": ` but a reference PR is something like https://github.com/babel/babel/pull/5889/files